chainerrl/agents/dqn.py

```diff
@@ -207,18 +207,21 @@ def call_test(self_, x):
     def update(self, experiences, errors_out=None):
         """Update the model from experiences
 
-        This function is thread-safe.
         Args:
-          experiences (list): list of lists of dicts.
-          The dict contains
-            state: cupy.ndarray or numpy.ndarray
-            action: int [0, n_action_types)
-            reward: float32
-            is_state_terminal: bool
-            next_state: cupy.ndarray or numpy.ndarray
-            weight (optional): float32
+            experiences (list): List of lists of dicts.
+                For DQN, each dict must contains:
+                  - state (object): State
+                  - action (object): Action
+                  - reward (float): Reward
+                  - is_state_terminal (bool): True iff the state is terminal
+                  - next_state (object): Next state
+                  - weight (float, optional): Weight coefficient. It can be
+                    used for importance sampling.
+            errors_out (list or None): If set to a list, then TD-errors
+                computed from the are appended to the list.
+
         Returns:
-          None
+            None
```

Comment:
Not too familiar with the docstring formatting, this tab shouldn't matter right?

Reply:
This is not a tab but two spaces. Google style docstring uses 4 spaces for an indentation. https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html